chain/actors/builtin/datacap/actor.go.template

```diff
@@ -0,0 +1,57 @@
+package datacap
+
+import (
+    "golang.org/x/xerrors"
+
+    "github.com/filecoin-project/go-address"
+    "github.com/filecoin-project/go-state-types/abi"
+    actorstypes "github.com/filecoin-project/go-state-types/actors"
+    builtin{{.latestVersion}} "github.com/filecoin-project/go-state-types/builtin"
+    "github.com/filecoin-project/go-state-types/cbor"
+
+    "github.com/filecoin-project/lotus/chain/actors"
+    "github.com/filecoin-project/lotus/chain/actors/adt"
+    "github.com/filecoin-project/lotus/chain/types"
+)
+
+var (
+	Address = builtin9.DatacapActorAddr
+	Methods = builtin9.MethodsDatacap
```

Comment:
Probably want `.latestVersion` here if we're not importing v9 explicitly

Reply:
Yes, definitely -- thank you!